civis/io/_tables.py

```diff
@@ -281,6 +283,107 @@ def f(x):
     return poll
 
 
+def civis_to_multifile_csv(sql, database, job_name=None, api_key=None,
+                           credential_id=None, include_header=True,
+                           compression='none', delimiter='|',
+                           unquoted=False, prefix=None,
+                           polling_interval=_DEFAULT_POLLING_INTERVAL,
+                           hidden=True):
+    """Unload the result of SQL query and return presigned urls.
+
```

Comment:
Might be nice to have a little explanation why you should use this function over `civis_to_csv`